public/scripts/wiki.js

```diff
@@ -131,15 +234,18 @@ var ArticleList = React.createClass({
     var parentThis = this;
     var articleTitles = this.props.articles.map(function(article, index){
       return (
-        <li key={index} onClick={parentThis.handleClick.bind(this, article._id)}> {article.title} </li>
+        <div className="article-list-item" key={index} onClick={parentThis.handleClick.bind(this, article._id)}> {article.title} </div>
       );
     });
 
     return (
       <div className="article-list">
-        <ul>  
-          {articleTitles}
-        </ul>
+        <h3><center>Users</center></h3>
+        <div className="article-list-item">Bobby McFingerson</div>
+        <div className="article-list-item">True dads TM</div>
```

Comment:
?I added it to login_react!! =) It'll get the users and put them in!


________________________________
From: Nora Mohamed <notifications@github.com>
Sent: Wednesday, March 2, 2016 8:39 PM
To: nmohamed/wiki-wiki
Cc: Yun-Hsin (Cynthia) Chen
Subject: Re: [wiki-wiki] Style (#2)


In public/scripts/wiki.js<https://github.com/nmohamed/wiki-wiki/pull/2#discussion_r54824491>:

>        );
>      });
>
>      return (
>        <div className="article-list">
> -        <ul>
> -          {articleTitles}
> -        </ul>
> +        <h3><center>Users</center></h3>
> +        <div className="article-list-item">Bobby McFingerson</div>
> +        <div className="article-list-item">True dads TM</div>


oops, forgot to get rid of that. I added that in case we wanted to show new users and to also see my changes without having to wait for the page to update. if you think it's not too much to add, then go for it!

-
Reply to this email directly or view it on GitHub<https://github.com/nmohamed/wiki-wiki/pull/2/files#r54824491>.